xds/internal/balancer/edsbalancer/eds_test.go

```diff
@@ -70,15 +68,38 @@ var (
 	}
 )
 
-func subConnFromPicker(p balancer.Picker) func() balancer.SubConn {
-	return func() balancer.SubConn {
-		scst, _ := p.Pick(balancer.PickInfo{})
-		return scst.SubConn
-	}
+func init() {
+	balancer.Register(bb{})
 }
 
 type s struct {
 	grpctest.Tester
+
+	cleanup func()
+}
+
+func (ss s) Setup(t *testing.T) {
+	ss.Tester.Setup(t)
+	// FIXME: delete this!!!
```

Comment:
Delete this and do something else? Or when can this be deleted?

Reply:
This was needed before the attributes xdsclient change.
Delete.